man/coredumpctl.xml

```diff
@@ -119,6 +119,14 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-d</option> <replaceable>DBG</replaceable></term>
+        <term><option>--debugger=</option><replaceable>DBG</replaceable></term>.
+
+        <listitem><para>Use the given debugger for the <command>debug</command>
+        command. Must be either gdb or lldb. </para></listitem>
```

Comment:
uh, it doesnt have to be either of those, people can actually specify whatever they want here, no? I mean if people like they could for example use "gnu ddd" instead here and things should just work fine, no?

Reply:
I think mentioning "gdb" or "lldb" as examples makes a ton of sense though, though should be enclosed in `<command>`

Reply:
also, this really should say that gdb is the implied default